### PR TITLE
Banners: professional learning 2019 for homepage, teacher /home & /courses

### PR DESCRIPTION
**Pull request by breville**
### code.org/
screenshot 2019-01-11 14 15 59

### teacher /home
screenshot 2019-01-14 10 19 26

### teacher /courses
screenshot 2019-01-14 10 20 43

Additional images are included for potential a/b testing.

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -1219,6 +1219,8 @@
   "specialAnnouncementDescriptionHoc2018Mc": "Sign up your class to try one of the over 150 NEW activities for the Hour of Code (Dec 3-9). Be sure to check out our brand new Minecraft activity for 2018, the Voyage Aquatic! Sign up your class today.",
   "specialAnnouncementHeadingPostHoc2018": "Go beyond an Hour of Code",
   "specialAnnouncementDescriptionPostHoc2018": "An hour is just the beginning. We offer comprehensive curriculum for grades K-12, professional learning programs including local workshops, and resources to help you and your school get started.",
+  "specialAnnouncementHeadingJoinProfessionalLearning2019": "Sign up for Professional Learning",
+  "specialAnnouncementDescriptionJoinProfessionalLearning2019": "98% of teachers recommend our engaging programs from elementary school to AP computer science for high school. Join our hands on workshops with other teachers from your area. Spaces are limited, so let us know if you're interested.",
```
Comment (Erin007) [inline apps/i18n/common/en_us.json]: hands-on?

File: apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx [hunk 2]
```diff
@@ -21,14 +21,22 @@ const styles = {
...
     color: color.white
   },
+  subHeadingSmallFont: {
+    paddingRight: 0,
+    paddingBottom: 20,
+    fontSize: 25,
+    lineHeight: 1.2,
+    fontFamily: '"Gotham 7r", sans-serif',
+    color: color.white
+  },
```
Comment (Erin007) [inline apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx:39]: We could extract the common styles between `subHeading` and `subHeadingSmallFont` and then set the font size only with the conditional below, but I don't feel strongly about it.

**Review by Erin007 — APPROVED** [review]
Can you check a language with longer strings (maybe Italian or Russian)?  I think with the smaller font size it should be fine, but would be good to double check.

**Comment by breville** [comment]
@Erin007 good point.  We since decided to make this announcement en-only on /home, as it already was on /courses.

**Pull request merged**